modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,6 +1,17 @@
 :description: This page lists all changes to status codes per Neo4j version.
 = Changes to status codes per Neo4j version
 
+== Neo4j 5.27
```

Comment:
No clue which version to put here. It's a Cypher25 feature currently pointing at dev (5.27? 2025.1? Cypher25?)